pages/desktop/frontend/details.py

```diff
@@ -733,6 +746,13 @@ def wait_for_rating_form(self):
                 expected.element_to_be_clickable(self._write_review_button_locator)
             )
 
+        def wait_for_delete_link(self):
+            self.wait.until(
+                expected.visibility_of_element_located(
+                    (By.CSS_SELECTOR, '.AddonReviewCard-delete')
+                )
+            )
+
```

Comment:
We already have a method which deals with the delete rating link (based on the locator you used).

Delete your method and, if needed, try to integrate it in the method we already have:

```
        @property
        def delete_rating_link(self):
            # waits for the ratings stars to be fully loaded and editable
            self.wait.until(
                expected.invisibility_of_element_located(
                    self._loaded_rating_stars_locator
                )
            )
            return self.find_element(*self._delete_rating_link_locator)
```

Reply:
Ok